tools/crm_resource.c

```diff
@@ -775,9 +775,6 @@ static GOptionEntry addl_entries[] = {
       "Force the action to be performed. See help for individual commands for\n"
       INDENT "additional behavior.",
       NULL },
-    { "host-uname", 'H', G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_STRING, &options.host_uname,
-      NULL,
-      "HOST" },
```

Comment:
Omit this and add a comment that it is used by resource-agents <4.2.0 (2018-10-24)

Reply:
Where specifically? I grepped for it in resource-agents v3.1.0, v3.9.0, v4.0.0, v4.1.0, and v4.2.0.